go/docs/_utils/pyproject_template.toml

```diff
@@ -0,0 +1,24 @@
+[tool.poetry]
```

Comment:
This file can be deleted. It's the template that has already been copied in https://github.com/scylladb/care-pet/pull/4/files#diff-522adf759addbd3b193c74ca85243f7d